Common/DataModel/OccupancyTables.h

```diff
@@ -0,0 +1,240 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2::aod
+{
+namespace occp
+{
+DECLARE_SOA_COLUMN(TfId, tfId, int64_t);
+DECLARE_SOA_COLUMN(BCsInTFlist, bcsInTFlist, std::vector<int64_t>);
+
+DECLARE_SOA_COLUMN(Occ_Prim_Unfm_80, occ_Prim_Unfm_80, std::vector<float>);
```

Comment:
Clarification: are the vector sizes variable or fixed? In the latter case one can also declare these columns as `float[N]`.

Reply:
It is variable, because BCsInTFlist  contain the list of BCs which participated  in calculation of occupancy table.
Occ_Prim_Unfm_80 is a fixed size vector. I will fix its size.

Reply:
In future, we will change the BC grouping to other values for checking. so it needs to be vector